src/core/anchor-expander.js

```diff
@@ -45,7 +47,8 @@ export function run() {
       default: {
         a.textContent = a.getAttribute("href");
         const msg = "ReSpec doesn't support expanding this kind of reference.";
-        showInlineError(a, msg, `Can't expand "#${id}".`);
+        const title = `Can't expand "#${id}".`;
+        pub("error", new RsError(msg, name, { title }));
```

Comment:
can we pass the `a`?

Reply:
Can't believe I missed this one after 3 self-reviews!